index.html

```diff
@@ -12,291 +12,265 @@
 
-    
-        <section class="documentation">
-            <div class="wrt">Watch our entire trip documentation</div>
-            <img src="Images/Video.png" alt="">
-        </section>
-  
+                    <div class="right-btn">
+                        Discover More
+                    </div>
+                </div>
+            </div>
+        </div>
 
-    <!-- END OF DOCUMENTATION SECTION -->
+        <div class="right">
+            <img src="Images/ImageGroup.png" alt="">
+        </div>
+    </section>
+    
+    <section class="documentation">
+        <div class="wrt">Watch our entire trip documentation</div>
+        <img src="Images/Video.png" alt="">
+    </section>
 
     <footer>
-        <div class="wrapper">
-            <div class="heading">
-                <div class="container">
-                    <h4>Get our best offer now!</h4>
-                    
-                    <div class="button">
-                        <a href="#">Click here!</a>
-                    </div>   
-                </div>
-                <img src="Images/footer-divider.png" alt="">
+        <div class="heading">
+            <div class="container">
+                <h4>Get our best offer now!</h4>
+                
+                <div class="button">
+                    <a href="#">Click here!</a>
+                </div>   
             </div>
+            <img src="Images/footer-divider.png" alt="" class="divider">
+        </div>
 
-            <div class="travel-footer">
-                <div class="left">
-                    <div class="content">
-                        <div class="h4">Travel</div>
-                        <div class="p">The best travel agent in the world. Contact us immediatly now!</div>
+        <div class="travel-footer">
+            <div class="left">
+                <div class="content">
+                    <div class="h4">Travel</div>
+                    <div class="p">The best travel agent in the world. Contact us immediatly now!</div>
 
-                        <div class="social-links">
-                            <a href="#">
-                                <img src="Images/IG.png" alt="">
-                            </a>
-                            <a href="#">
-                                <img src="Images/twitter.png" alt="">
-                            </a>
-                            <a href="#">
-                                <img src="Images/YT.png" alt="">
-                            </a>
-                        </div>
+                    <div class="social-links">
+                        <a href="#">
+                            <img src="Images/IG.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="Images/twitter.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="Images/YT.png" alt="">
+                        </a>
```

Comment:
(OPTIONAL)

It is best practice to add a title attribute to all anchor tags. For example, 

`<a href="#" title="this is an anchor tag"></a>`